pkgs/tools/virtualization/nixos-nspawn/nixos-nspawn.py

```diff
@@ -0,0 +1,510 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p python3 -i python3 python3Packages.rich
```

Comment:
```suggestion
#!nix-shell -i python3 -p "python3.withPackages(ps: with ps; [ rich ])"
```

we want a python env with rich, not python and python3.pkgs.rich.